definition_builder.go

```diff
@@ -205,18 +205,28 @@ func (b definitionBuilder) buildProperty(field reflect.StructField, model *spec.
 	}
 
 	// not a primitive
+	// Need to re-assign the `example` field, because the `prop` variable is a new one, different than the `prop` variable
```

Comment:
i think this comment line is not clear as it says `prop` twice

Reply:
right, let me try to make it more concise. Thanks for the feedback!